feat: update field:type matching for an interface

senaki opened the pull request:
Update rule to catch field:type even if field is within double quote and contains ':'. Prevent parsing line starting with import keyword.

ad-on-is commented:
Could you give me a short example, what usecase this covers? I assume one would import something into an interface-file, shouldn't that line be ignored then?

```
import FooType from 'foo'

export default BarInterface {
foo: FooType
}
```

senaki commented:
> Could you give me a short example, what usecase this covers? I assume one would import something into an interface-file, shouldn't that line be ignored then?
> 
> This is how I understood it
> 
> ```
> import FooType from 'foo'
> 
> export default BarInterface {
> foo: FooType
> }
> ```

An exemple is : 
```
import toto from "toto"
export default interface tata {
"aPossibleNotation":toto
 "anOtherPossible:Name":type
}
```
In current implementation, `s=line.split(":"); field=s[0]; type=s[1]` would lead to `field ="anOtherPossible" ; type="Name"` instead of of `field="anOtherPossible:Name"` and `type="type"` as expected for the last field. 

Using regex groups allows to fix this. 

I have added the lines starting with "import" keyword to the ignore list in foreach callback since it is not relevant in the interface definition as comment line. If not the case, the regex may match `"toto"` in my example. 

It may be improved.

ad-on-is commented:
Wow... I've never seen such a notation

```js
interface tata {
"aPossibleNotation":String
 "anOtherPossible:Name":integer
}
```

senaki commented:
> Wow... I've never seen such a notation
> 
> ```js
> interface tata {
> "aPossibleNotation":String
>  "anOtherPossible:Name":integer
> }
> ```

Yeah, it's however, it's not so rare to encounter this, and it is a notation allowed by the standard. Take a look at the [official documentation](https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types-with-template-strings) to see by yourself.
